cfgov/unprocessed/css/molecules/pagination.less

```diff
@@ -0,0 +1,139 @@
+                          number out of {{ value.pages }} total pages
+                      </span>
+                  </label>
+                  <input class="m-pagination_current-page"
+                         id="pagination_current-page"
+                         name="pagination_current-page"
+                         type="number" min="1" max="149"
+                         value="149">
+                  <span class="m-pagination_label">
+                      <span aria-hidden="true">
+                          of 149
+                      </span>
+                  </span>
+                  <button id="m-pagination_submit"
+                          type="submit">
+                      Go
+                  </button>
+              </form>
+          </nav>
+  tags:
+    - cfgov-molecules
+*/
+
+@pagination-bg-color: #F0F1F1;
+@pagination-text-color: #75787B;
+@pagination-font-size__px: 16px;
+@pagination-font-size__em: unit( @pagination-font-size__px / @base-font-size-px, em );
+@pagination-height__px: 42px;
+
+.m-pagination {
+    display: block;
+    position: relative;
+    height: @pagination-height__px;
+
+    label, &_label {
+        display: inline-block;
+        margin: 0;
+       .webfont-regular();
+        font-size: @pagination-font-size__em;
+        vertical-align: middle;
+    }
+
+    input[type="number"] {
+      .u-inline-block();
+      // We need enough room for three digits and the number spinners for type="number" inputs.
+      width: unit( 42px / @pagination-font-size, em );
+      margin: 0 @pagination-font-size__em / 2;
+      font-size: @pagination-font-size__em;
+      .webfont-medium();
+      text-align: right;
+      vertical-align: middle;
+    }
+
+    button[type="submit"] {
+        .u-inline-block();
+        margin: 0 0 0 @pagination-font-size__em / 2;
+        font-size: @pagination-font-size__em;
+        .webfont-medium();
+        vertical-align: middle;
+    }
+
+    form {
+        display: block;
+        padding: unit( 5px / @base-font-size-px, em );
+        background: @pagination-bg-color;
+        border-radius: unit( 4px / @base-font-size-px, em );
+        color: @pagination-text;
+        text-align: center;
+    }
+
+    &_btn-prev,
+    &_btn-next {
+      .webfont-medium();
+      height: @pagination-height__px;
+      font-size: @pagination-font-size__em;
+      margin-bottom: @pagination-font-size__em / 2;
+
+      &.btn__disabled {
+        background-color: @pagination-bg-color;
+      }
+    }
+
+    &_btn-next {
+        position: absolute;
+        top: 0;
+        right: 0;
+    }
+
+    .respond-to-min(@bp-sm-min, {
+
+        &_btn-prev {
+            position: absolute;
+            top: 0;
+            left: 0;
+
+            &,
+            &:link,
+            &:visited {
+                border-top-right-radius: 0;
+                border-bottom-right-radius: 0;
+            }
+        }
+
+        &_btn-next {
+          &,
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/e2dacda37545a885a64b835357988642043605f8